### PR TITLE
enable build with badgerdb

### PR DESCRIPTION
**Pull request by gotjoshua**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -87,6 +87,9 @@ endif
 ldflags += $(LDFLAGS)
 ldflags := $(strip $(ldflags))
 
+build_tags += $(BUILD_TAGS)
+build_tags := $(strip $(build_tags))
```
Comment (robert-zaremba) [inline Makefile:91]: why changing the order?

Comment (gotjoshua) [reply Makefile:91]: @robert-zaremba  so that the flags actually get in (instead of overwritten).

Would be great if this could get merged with 0.6.2

Comment (gotjoshua) [reply Makefile:91]: the boltdb tags also get ignored in the old order...

**Review by robert-zaremba — APPROVED** [review]
Good, thanks for contribution!

**Pull request merged**